lib/locale/pl/strings.php

```diff
@@ -24,6 +24,7 @@
  *  $Id$
  */
 
+$_LANG['Only verifier can change this now'] = 'Tylko weryfikator może to zmienić w tej chwili';
```

Comment:
To jakoś dziwnie brzmi - nie lepiej byłoby:
Only verifier can change this (bez now)?
i po polsku:
Tylko weryfikator może to zmienić.

Reply:
Poprawione.